README.md

```diff
@@ -4,26 +4,75 @@ The official [shareable ESLint config](http://eslint.org/docs/developer-guide/sh
 
 ## Usage
 
----
-
 We export three ESLint configurations.
 
-### eslint-config-cesium
+### eslint-config-cesium default
+
+This config contains basic Cesium syntax and style config, from which `browser` and `node` extend. Extends `@eslint/js` recommended rules and `eslint-config/prettier` with additional rules.
+
+[`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files)
+
+```js
+import configCesium from "eslint-config-cesium";
 
-This config contains basic Cesium syntax and style config, from which `browser` and `node` extend. Extends `eslint:recommended` and `prettier` with additional rules.
+exports default [
+    // Apply recommended rules to all files
+    configCesium.configs.recommended,
+    // Apply recommended rules to JS files with an override
+    {
+        files: ["**/*.js"],
+        rules: {
+            ...configCesium.configs.recommended.rules,
+            "no-unused-vars": "off"
+        } 
+    },
+]
+```
 
 ### eslint-config-cesium/browser
 
 For use in browser environments.
 
+[`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files)
+
+```js
+import configCesium from "eslint-config-cesium";
+
+exports default [
+    // Apply recommended rules to all files
+    configCesium.configs.browser,
+    // Apply recommended rules to JS files with an override
+    {
+        files: ["**/*.js"],
+        ...configCesium.configs.browser,
+        rules: {
+            ...configCesium.configs.browser.rules,
```

Comment:
Is this necessary? is this the recommended way of doing it? It seems weird to require multiple spreads of our config. I though the cascading nature of the new flat config wouldn't require stuff like this?

Reply:
@ggetz can verify, but I believe the problem here is the use of the spread operator has non-intuitive mechanics for how it's used.

the `rules` object specified here completely overwrites the `rules` object from `configCesium.configs.browser`, so if you want to keep the rules, you need the additional spread here.

Reply:
For my own app, I end up not going this and instead have an array of rules, which we could also do here if we wanted. I don't think there's a perfect answer.

Reply:
Right, but is it better style to do something like this? Just a genuine question about what the expected style is from eslint if we have any idea

```js
exports default [
    // Apply recommended rules to JS files with an override
    {
        files: ["**/*.js"],
        ...configCesium.configs.browser,
    },
    {
        files: ["**/*.js"],
        rules: {
            "no-unused-vars": "off",
        }
    }
]
```

Reply:
Either will work. I don't think there is an expected style either, at least not from what I've seen.